vertical-pod-autoscaler/pkg/utils/vpa/capping.go

```diff
@@ -130,6 +130,7 @@ func getCappedRecommendationForContainer(
 				cappingAnnotations = append(cappingAnnotations, annotations...)
 			}
 		}
+		klog.V(3).Infof("processed capped recommendations for %s: %v", container.Name, cappingAnnotations)
```

Comment:
I assume you're expecting to see here messages that indicate a recommendation was capped to the `minAllowed` value defined in the VPA. However, I think this will not be the case, as [the recommender itself already does cap the `target`](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/recommender/routines/capping_post_processor.go#L35) to the currently defined `minAllowed` value when writing a new recommendation to the VPA.

If you're interested in the difference between actually computed recommendation and the value in `target`, you have to compare `uncappedTarget` and `target` in the VPA status, e.g. by constantly exporting them using kube-state-metrics.

As this log line won't do much, I propose removing it from this PR.

Reply:
Sounds good, thank you! I see now that focusing on the (capped) `target` and `uncappedTarget` in the other log achieves the same result 👍